sklearn/svm/base.py

```diff
@@ -604,30 +603,24 @@ def _get_liblinear_solver_type(multi_class, penalty, loss, dual):
       - loss
       - dual
 
-    The same number is internally by LibLinear to determine which
-    solver to use.
+    The same number is also internally used by LibLinear to determine
+    which solver to use.
     """
-
-    # nested dicts containing level 1: available loss functions, 
+    # nested dicts containing level 1: available loss functions,
     # level2: available penalties for the given loss functin,
     # level3: wether the dual solver is available for the specified
     # combination of loss function and penalty
     _solver_type_dict = {
-        'logistic_regression': {
-            'l1': {False: 6},
-            'l2': {False: 0, True: 7}},
-        'hinge' : {
-            'l2' : {True: 3}},
-        'squared_hinge': {
-            'l1': {False : 5},
-            'l2': {False: 2, True: 1}},
-        'epsilon_insensitive': {
-            'l2': {True: 13}},
-        'squared_epsilon_insensitive': {
-            'l2': {False: 11, True: 12}},
+        'logistic_regression': {'l1': {False: 6},
+                                'l2': {False: 0, True: 7}
+                                },
+        'hinge': {'l2': {True: 3}},
+        'squared_hinge': {'l1': {False: 5},
+                          'l2': {False: 2, True: 1}},
+        'epsilon_insensitive': {'l2': {True: 13}},
+        'squared_epsilon_insensitive': {'l2': {False: 11, True: 12}},
```

Comment:
You mean the formatting of the nested dict? I agree I preferred the old formatting.